pkg/binding/bindingpolicy.go

```diff
@@ -88,6 +88,47 @@ func (c *Controller) syncBindingPolicy(ctx context.Context, bindingPolicyName st
 		logger.V(5).Info("Enqueued Binding for syncing, while handling BindingPolicy", "name", bindingPolicy.Name)
 		c.enqueueBinding(bindingPolicy.GetName())
 
+		statusCollectorName, err := c.identifyMissingStatusCollector(bindingPolicy)
```

Comment:
Why not identify all the missing StatusCollector objects? It would be more helpful to a user to reveal all the problems.